lib/push-jenkins-update.js

```diff
@@ -13,6 +14,10 @@ function pushStarted (options, build, cb) {
 
   const optsWithPr = Object.assign({ pr }, options)
 
+  if (build.identifier === 'node-test-pull-request') {
+    createPrComment(Object.assign({ number: pr }, options), `CI: ${build.url}`)
```

Comment:
```suggestion
	let message = ''
	switch(build.status) {
		case 'pending':
		    message = `CI: ${build.url}`)
		case 'failure':
			message = 'CI job failed.'
		case 'success':
		case 'unstable':
			message = 'CI job passed.'
		default:
			// ¯\\_(ツ)_/¯
			return;
	}
    createPrComment(Object.assign({ number: pr }, options), message)
```

We get at least two messages for each job.
`'pending'` when the job starts. And then the final status.

Reply:
FYI I wrote this via GitHub so 90% chance this has syntax error, consider this pseudo-code.

Reply:
Huh, good suggestion.

Currently the bot handles [starting](https://github.com/nodejs/github-bot/blob/master/scripts/jenkins-status.js#L25) and [ending](https://github.com/nodejs/github-bot/blob/master/scripts/jenkins-status.js#L54) of jobs separately. The changes proposed in this PR only affects jobs that has just _started_, not _ended_ like the failures/success/unstable statuses.

Previously I've gotten the impression there are concerns from collaborators about the bot "spamming" PRs with comments. Is that something to consider before introducing what you suggest here, or have collaborators been wanting this for a while already?

Btw reducing the duplication seen handling starting & ended jobs surely sounds like a good fit for a later PR though.

Reply:
__Personally__ I'd love to have the "ending" message... It will trigger a notification which is something that will help my workflow. a la https://github.com/nodejs/build/issues/1502.

I know it's probably written somewhere that we should track when we start a CI job, but GitHub already does that via the "status", so IDK.

/Ping @Trott 

Anyway I'd think at minimum we should:

```suggestion
    if (build.identifier === 'node-test-pull-request' && build.status === 'pending) {
```

Reply:
Adding that `build.status === 'pending'` sounds good for now IMO. It'll also serve as an explicit reminder if the handling of starting & ending jobs gets refactored into one later.